integration-cli/docker_api_logs_test.go

```diff
@@ -85,3 +86,94 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func extractBody(c *check.C, url string) []string {
```

Comment:
Yeah, but I'm using it repeatedly for some equality tests - so I don't see any harm in adding it as a utility fn.